substrate/frame/balances/src/benchmarking.rs

```diff
@@ -31,6 +31,14 @@ const SEED: u32 = 0;
 // existential deposit multiplier
 const ED_MULTIPLIER: u32 = 10;
 
+fn minimum_balance<T: Config<I>, I: 'static>() -> T::Balance {
+	if cfg!(feature = "insecure_zero_ed") {
+		100u32.into()
```

Comment:
this is not really correct, min balance is `0` in this case. I would rather fix specifically failing bench/es. this might lead to a wrong benchmark at least with `insecure_zero_ed` feature enabled

Reply:
I didn't check all of them, but most of them are just using this to set some balance or whatever. I don't think there is any problem in using `100` if the ED is zero.